pkg/controller/configmap/configmap_controller.go

```diff
@@ -76,7 +100,8 @@ func add(mgr manager.Manager, sharedInfo *sharedinfo.SharedInfo, r reconcile.Rec
 	}
 
 	// Watch for changes to primary resource ConfigMap
-	err = c.Watch(&source.Kind{Type: &corev1.ConfigMap{}}, &handler.EnqueueRequestForObject{})
+	// The configmap resource changes may appear in mutiple namespaces, only changes from watched namespace of operator will be queued for handler
+	err = c.Watch(&source.Kind{Type: &corev1.ConfigMap{}}, &handler.EnqueueRequestForObject{}, byNameSpaceFilter(operatortypes.OperatorNamespace))
```

Comment:
Below is the way to get watchedNamespace, do not use operatortypes.OperatorNamespace directly.
```
	watchedNamespace := r.status.OperatorNamespace
	if watchedNamespace == "" {
		log.Info(fmt.Sprintf("ConfigMapController can only watch a single namespace, defaulting to: %s",
			operatortypes.OperatorNamespace))
		watchedNamespace = operatortypes.OperatorNamespace
	}
```

Reply:
Thanks a lot for advice.